packages/cli/src/index.ts

```diff
@@ -764,6 +779,78 @@ yargs
       commandWrapper(handlerBuildCommand)(argv as Arguments<BuildDocsArgv>);
     }
   )
+  .command(
+    'translations <locales...>',
+    'Creates or updates translations.yaml files and fills them with missing built-in translations and translations from the redocly.yaml and sidebars.yaml files.',
+    (yargs) =>
+      yargs
+        .positional('locales', {
```

Comment:
This should be singular, the description is a singular code and that is the expected behaviour. We already had to amend some other command where we tried to allow multiple values to a positional argument and it caused confusion!